docs/source/tutorials/tutorial.ipynb

```diff
@@ -804,7 +788,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Beyond the collection of image files demonstrated here, you can alternatively run CleanVision on: [Hugging Face datasets](https://github.com/cleanlab/cleanvision/blob/main/docs/source/tutorials/huggingface_dataset.ipynb) and [torchvision datasets](https://github.com/cleanlab/cleanvision/blob/main/docs/source/tutorials/torchvision_dataset.ipynb).**"
+    "Beyond the collection of image files demonstrated here, you can alternatively run CleanVision on: [Hugging Face datasets](https://github.com/cleanlab/cleanvision/blob/main/docs/source/tutorials/huggingface_dataset.ipynb) and [torchvision datasets](https://github.com/cleanlab/cleanvision/blob/main/docs/source/tutorials/torchvision_dataset.ipynb)."
```

Comment:
the formatting here was broken on our website:

![Screen Shot 2023-11-16 at 7 59 15 PM](https://github.com/cleanlab/cleanvision/assets/1390638/a187ba38-dfd5-4333-ad0e-92d000b0fba4)

Maybe you have better ideas on how to make this even better looking